### PR TITLE
Addition of Value type supported

### PR DESCRIPTION
**Pull request by joinimran**
Added information on value type supported and example.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8402

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/client-management/mdm/policy-csp-timelanguagesettings.md [hunk 1]
```diff
@@ -78,7 +78,8 @@ Specifies the time zone to be applied to the device. This is the standard Window
 
 <!--/Description-->
 <!--SupportedValues-->
-
+Value type is String. Supported values:
+- Name of Standard Time Zone - e.g. Pacific Standard Time, Mountain Standard Time
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-timelanguagesettings.md:82]:
```suggestion
- Name of Standard Time Zone - for example, Pacific Standard Time, Mountain Standard Time. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by illfated — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Pull request merged**